README.md

```diff
@@ -105,14 +105,17 @@ The CLI argument `--config` is not required.
 If it is not provided, `markdownlint-cli` looks for the file `.markdownlint.json`/`.markdownlint.yaml`/`.markdownlint.yml` in current folder, or for the file `.markdownlintrc` in the current or all parent folders.
 The algorithm is described in detail on the [`rc` package page][rc-standards].
 If the `--config` argument is provided, the file must be valid JSON, JSONC, JS, or YAML.
-JS configuration files contain JavaScript code, must have the `.js` extension, and must export (via `module.exports = ...`) a configuration object of the form shown above.
+JS configuration files contain JavaScript code, must have the `.(c)js` extension, and must export (via `module.exports = ...`) a configuration object of the form shown above.
```

Comment:
Nice. For clarity, could you change to ".js or .cjs" so people don't need to understand parentheses?

Reply:
Changed.